tests/unit/test_driven.py

```diff
@@ -28,7 +28,7 @@ def test_minimize_distance_no_growth_rate(iJO1366):
             'namespace': 'bigg.reaction',
             'measurements': [2.36, 2.45, 1.92]
         }]
-        minimize_distance(iJO1366, biomass_reaction, measurements)
+        minimize_distance(iJO1366, biomass_reaction, None, measurements)
```

Comment:
Should there also be a test with a growth rate?

Reply:
Yes that would be a good idea, I'll add an endpoint test.